### PR TITLE
Force funnel link to reload page

### PR DESCRIPTION
**Pull request by alexey-petrov**
Ref: https://doesitsparkle.slack.com/archives/C01H9D1P1PE/p1647626388615959

Force funnel banner links will now reload the page

### CODE REVIEW

**Comment by alexey-petrov** [comment]
Closed in favor of https://github.com/sparkletown/sparkle/pull/2938

File: src/components/atoms/LinkButton/LinkButton.tsx [hunk 1]
```diff
@@ -11,23 +14,28 @@ export interface LinkButtonProps
     HTMLAnchorElement
   > {
   href: string;
+  isBlank: boolean;
```
Comment (goran-peoski-work) [inline src/components/atoms/LinkButton/LinkButton.tsx:17]: I can't find a place with `<LinkButton isBlank />`

**Review by goran-peoski-work — APPROVED** [review]
LGTM

File: src/components/atoms/LinkButton/LinkButton.tsx (hunk 1 above)
Comment (alexey-petrov) [reply src/components/atoms/LinkButton/LinkButton.tsx:17]: it's right here https://github.com/sparkletown/sparkle/blob/d592e3dfb39e1927aca1965a5d324915ee8c3de9/src/components/molecules/AnnouncementMessage/AnnouncementMessage.tsx

**Pull request merged**